build/platform_and_feature_flags.props

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNetCore)' ">
     <DefineConstants>$(DefineConstants);NET_CORE;SUPPORTS_CONFIDENTIAL_CLIENT;SUPPORTS_CUSTOM_CACHE;SUPPORTS_BROKER;SUPPORTS_WIN32;</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == '$(TargetFrameworkNet6)' ">
+    <DefineConstants>$(DefineConstants);NET_CORE;SUPPORTS_CONFIDENTIAL_CLIENT;SUPPORTS_CUSTOM_CACHE;SUPPORTS_WIN32;</DefineConstants>
```

Comment:
Should there be a `SUPPORTS_SYSTEM_TEXT_JSON` flag?

Reply:
Hmm, I think SUPPORTS_SYSTEM_TEXT_JSON is better. Because currently STJ reference is added for net6 target, not Maui. But NET6_OR_GREATER may get triggered on net6-maui and throw a reference not found exception.

Reply:
Hmm, net6 doesn't support old broker because of windows dependencies, kind of like netcoreapp3.1. We would have to add net6-windows, like we do for net5-win target. Net6 should support new WAM broker. I removed the broker flag just to make it easier - not to run any broker tests, since they are run for netcoreapp3.1 tests.